tests/TestClasses/TestJSON.php

```diff
@@ -14,7 +14,7 @@ class TestJSON implements \JsonSerializable
      * which is a value of any type other than a resource.
      * @since 5.4.0
      */
-    public function jsonSerialize()
+    public function jsonSerialize(): mixed
```

Comment:
This pseudo type is the only reason we are removing the php 7.* support?
Can it be avoided?
Why it can not be array?

Reply:
Right, I modified as you suggested